default.nix

```diff
@@ -80,9 +80,9 @@ let iosSupport = system == "x86_64-darwin";
         hackGetOverlay
         bindHaskellOverlays
         forceStaticLibs
+        splicesEval
         mobileGhcOverlay
         allCabalHashesOverlay
-        splicesEval
```

Comment:
The ordering ends up mattering on the other PR because we need to nuke the upstream patches from Nixpkgs that are merged into our fork.